src/app/index.scss

```diff
@@ -2013,8 +2013,8 @@ md-content.md-default-theme, md-content {
   FOOTER COMPONENT
 */
 
-.footer {
-  position: absolute;
+footer {
+//  position: absolute;
```

Comment:
why in a comment ? maybe should be removed?

Reply:
Because some issues keep popping up again in this codebase, I decided to use comments instead of deleting, in case we need to re-introduce them later. In case I am breaking something somewhere else. Feel free to delete it if you prefer